internal/service/federatedsettingsorgconfig/resource_federated_settings_connected_org_test.go

```diff
@@ -12,13 +13,25 @@ import (
 	"github.com/mongodb/terraform-provider-mongodbatlas/internal/testutil/acc"
 )
 
+func TestAccFederatedSettingsOrgCreate_createError(t *testing.T) {
```

Comment:
```suggestion
func TestAccFederatedSettingsOrg_createError(t *testing.T) {
```

Reply:
Thanks. Updated in [b4964e6](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/2290/commits/b4964e64fcf3b7e5694bb425719387c29179e7b4)